api/_hyperfunctions/candlestick_agg/examples.md

```diff
@@ -0,0 +1,210 @@
+the trading volume, for each 1 hour period in the last day:
+
+``` sql
+SELECT
+    time_bucket('1 hour'::interval, "time") AS ts,
+    symbol,
+    toolkit_experimental.open(toolkit_experimental.candlestick_agg("time", price, volume)),
+    toolkit_experimental.high(toolkit_experimental.candlestick_agg("time", price, volume)),
+    toolkit_experimental.low(toolkit_experimental.candlestick_agg("time", price, volume)),
+    toolkit_experimental.close(toolkit_experimental.candlestick_agg("time", price, volume)),
+    toolkit_experimental.volume(toolkit_experimental.candlestick_agg("time", price, volume))
+FROM stocks_real_time
+WHERE "time" > now() - '1 day'::interval
+GROUP BY ts, symbol
+;
+
+-- or
+
+WITH cs AS (
+    SELECT time_bucket('1 hour'::interval, "time") AS hourly_bucket,
+      symbol,
+      toolkit_experimental.candlestick_agg("time", price, volume) AS candlestick
+    FROM stocks_real_time
+    WHERE "time" > now() - '1 day'::interval
+    GROUP BY hourly_bucket, symbol
+)
+SELECT hourly_bucket,
+  symbol,
+  toolkit_experimental.open(candlestick),
+  toolkit_experimental.high(candlestick),
+  toolkit_experimental.low(candlestick),
+  toolkit_experimental.close(candlestick),
+  toolkit_experimental.volume(candlestick)
+FROM cs
+;
+```
+
+### Create a continuous aggregate from tick data and roll it up
+
+Create a continuous aggregate on your stock trade data:
+
+```sql
+CREATE MATERIALIZED VIEW candlestick
+WITH (timescaledb.continuous) AS
+SELECT time_bucket('1 minute'::interval, "time") AS ts,
+  symbol,
+  toolkit_experimental.candlestick_agg("time", price, volume) AS candlestick
+FROM stocks_real_time
+GROUP BY ts, symbol
+;
+```
+
+Query your by-minute continuous aggregate over stock trade data for the opening,
+high, low, and closing (OHLC) prices, along with their timestamps, in the last
+hour:
+
+``` sql
+SELECT ts,
+  symbol,
+    toolkit_experimental.open_time(candlestick),
+    toolkit_experimental.open(candlestick),
+    toolkit_experimental.high_time(candlestick),
+    toolkit_experimental.high(candlestick),
+    toolkit_experimental.low_time(candlestick),
+    toolkit_experimental.low(candlestick),
+    toolkit_experimental.close_time(candlestick),
+    toolkit_experimental.close(candlestick)
+FROM candlestick
+WHERE ts > now() - '1 hour'::interval
+;
+```
+
+Roll up your by-minute continuous aggregate into daily buckets and return the
+Volume Weighted Average Price for `AAPL` for the last month:
+
+``` sql
+SELECT time_bucket('1 day'::interval, ts) AS daily_bucket,
+    symbol,
+    toolkit_experimental.vwap(toolkit_experimental.rollup(candlestick))
+FROM candlestick
+WHERE symbol = 'AAPL'
+      AND ts > now() - '1 month'::interval
+GROUP BY daily_bucket
+ORDER BY daily_bucket
+;
+```
+
+Roll up your by-minute continuous aggregate into hourly buckets and return the
+the opening, high, low, and closing prices and the volume for each 1 hour period
+in the last day:
+
+``` sql
+SELECT ts,
```

Comment:
@rtwalker can you double-check this example? It may be that I'm not understanding the syntax, but it looks to me as if this isn't rebucketed by the hour?